### PR TITLE
Fix directive aliases in inner template

### PR DESCRIPTION
**Pull request by luwes**
This fixes an issue where the inner template aliases were not transformed and broke the directives.

### CODE REVIEW

File: src/js/utils/template-processor.js [hunk 1]
```diff
@@ -23,33 +23,47 @@ const operators = {
...
-            if (evaluateCondition(part.expression, state)) {
-              part.replace(
-                new TemplateInstance(part.template, state, processor)
-              );
-            } else {
-              part.replace('');
-            }
-            break;
-        }
+      if (part.directive && part instanceof InnerTemplatePart) {
```
Comment (cjpillsbury) [inline src/js/utils/template-processor.js:65]: 😎 I like this.

Comment (cjpillsbury) [reply src/js/utils/template-processor.js:65]: question(non-blocking): Should this conditional order be switched?

File: src/js/utils/template-processor.js [hunk 2]
```diff
@@ -23,33 +23,47 @@ const operators = {
...
+      part.replace(new TemplateInstance(part.template, state, processor));
+    } else {
+      part.replace('');
+    }
+  },
+};
+
+const DirectiveNames = Object.keys(Directives);
+
+export function transformDirectiveAliases(template) {
```
Comment (cjpillsbury) [inline src/js/utils/template-processor.js:47]: suggestion(non-blocking): I think `transform` functions tend to indicate (still potentially mutative) new objects (e.g. https://lodash.com/docs/4.17.15#transform or https://www.educative.io/answers/what-is-the-transform-function-in-cpp). Consider calling this `updateDirectiveAliases` or `modifyDirectAliases` or something like that? (I dunno, names are hard)

**Review by cjpillsbury — APPROVED** [review]
Only a few non-blocking comments. Otherwise, 👌

File: src/js/utils/template-processor.js (hunk 1 above)
Comment (luwes) [reply src/js/utils/template-processor.js:65]: I imagined a property lookup is faster than an instanceof check but not sure.
yes, I can see why it makes a little more sense in an ordering sense to first check instanceof but perf :)

**Pull request merged**